app/views/evaluations/_submissions_table.html.erb

```diff
@@ -1,41 +1,52 @@
 <%= render 'shared/assignment_stats', heading: 'Total Submissions', show_all_stats: false %>
 
-<table aria-label="List of Submissions to Evaluate" class="usa-table usa-table--stacked-header usa-table--borderless width-full gray-header margin-bottom-10">
+<table aria-label="Submissions assigned to me" class="usa-table usa-table--stacked-header usa-table--borderless width-full gray-header margin-bottom-10">
   <thead>
     <tr>
       <th scope="col">Submission ID</th>
-      <th scope="col">My evaluation status</th>
-      <th scope="col">My score</th>
+      <th scope="col">Evaluation Status</th>
+      <th scope="col">My Score</th>
       <th scope="col">Average Score</th>
-      <th scope="col"><span class="usa-sr-only">Evaluate</span></th>
+      <th scope="col">Action</th>
     </tr>
   </thead>
   <tbody>
     <% @assigned_submissions.each do |assignment| %>
-      <tr>
+      <tr class="border-y-2px border-primary" style="height: 4rem;">
         <th data-label="Submission ID" scope="row">
-          <span class="mobile-lg:display-none">Submission ID</span><%= assignment.submission.id %>
-        </th>
-        <td data-label="My evaluation status">
-          <div class="display-flex flex-align-center">
-            <span class="usa-tag padding-x-1 <%= evaluation_status_color(assignment.evaluation_status) %>">
+          <div class="margin-bottom-1 mobile-lg:margin-bottom-3">
+            <%= link_to materials_submission_path(assignment.submission), class: "text-primary" do %>
+              <span class="mobile-lg:display-none">Submission ID </span><%= assignment.submission.id %>
+            <% end %>
+          </div>
+        </th>        
+        <td data-label="Evaluation Status">
+          <div class="display-flex flex-align-center margin-bottom-2">
+            <span class="usa-tag padding-x-1 text-center width-15 <%= evaluation_status_color(assignment.evaluation_status) %>" style="min-width: 120px; display: inline-block;">
               <%= assignment.evaluation_status.to_s.titleize %>
             </span>
           </div>  
         </td>
-        <td data-label="My score">
-          <div class="display-flex flex-align-center">  
-            <%= evaluator_score(assignment).formatted_score %>
+        <td data-label="My Score">
+          <div class="display-flex flex-align-center margin-bottom-2">  
+            <% score = evaluator_score(assignment, format: :dash) %>
+            <% if score.raw_score.to_f > 0 %>
```

Comment:
I'd prefer to check the `assignment.evaluation_status == :completed` just to account for all the different states. the `evaluator_score` function is very obfuscated with conditionals and I can't easily tell what the value would be for different statuses so checking the evaluation_status itself seems more direct.
also what happens if the evaluator gave the submission a score of 0?